OOP/MenuNew.java

```diff
@@ -0,0 +1,16 @@
+
+public class MenuNew  {
```

Comment:
ACTION: Shouldn't this class be extending Main.java? I would make much sense and will be a better OOP strategy.